app/lucy/src/app/directives/element-focus/element-focus.directive.ts

```diff
@@ -0,0 +1,18 @@
+import { Directive, ElementRef, Input, OnInit } from '@angular/core';
+
+@Directive({
+  selector: '[appElementFocus]'
+})
+export class ElementFocusDirective implements OnInit {
```

Comment:
Add Documentation

Reply:
As `comments` or in the actual `README`?

Reply:
as general code comment